pyproject.toml

```diff
@@ -246,4 +252,5 @@ dev = [
     "mypy",
     "pyright",
     "pytest-cov>=6.0.0",
+    "types-PyYAML"  # Type stubs for PyYAML
```

Comment:
since we are using ruamel.yaml and not PyYAML, how does this help?

Reply:
Btw. there is a pending PR to support ruamel.yaml https://github.com/python/typeshed/pull/12584